sdk/remoterendering/azure-mixedreality-remoterendering/azure-mixedreality-remoterendering_python.json

```diff
@@ -0,0 +1 @@
oteRenderingClient.stop_rendering_session:async.param(session_id", "Value": "session_id"}, {"Kind": 3, "Value": ":"}, {"Kind": 2, "Value": " "}, {"Kind": 6, "Value": "str"}, {"Kind": 3, "Value": ""}, {"Kind": 3, "Value": ","}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "            "}, {"Kind": 5, "DefinitionId": "azure.mixedreality.remoterendering.aio.RemoteRenderingClient.stop_rendering_session:async.param(**kwargs", "Value": ""}, {"Kind": 0, "DefinitionId": "azure.mixedreality.remoterendering.aio.RemoteRenderingClient.stop_rendering_session:async.param(**kwargs", "Value": "**kwargs"}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "        "}, {"Kind": 3, "Value": ")"}, {"Kind": 2, "Value": " "}, {"Kind": 3, "Value": "->"}, {"Kind": 2, "Value": " "}, {"Kind": 5, "DefinitionId": "azure.mixedreality.remoterendering.aio.RemoteRenderingClient.stop_rendering_session:async.returntype", "Value": ""}, {"Kind": 6, "Value": "None"}, {"Kind": 3, "Value": ""}, {"Kind": 1, "Value": ""}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 1, "Value": ""}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}], "Diagnostics": [{"DiagnosticId": "AZ_PY_1", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.AssetConversion.__init__"}, {"DiagnosticId": "AZ_PY_2", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.AssetConversionInputSettings.__init__"}, {"DiagnosticId": "AZ_PY_3", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.ConversionOutput.__init__"}, {"DiagnosticId": "AZ_PY_4", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.AssetConversionOutputSettings.__init__"}, {"DiagnosticId": "AZ_PY_5", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.AssetConversionSettings.__init__"}, {"DiagnosticId": "AZ_PY_6", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.Error.__init__"}, {"DiagnosticId": "AZ_PY_7", "Text": "list API list_asset_conversions should return ItemPaged or AsyncItemPaged instead of ItemPaged[AssetConversion]", "TargetId": "azure.mixedreality.remoterendering.RemoteRenderingClient.list_asset_conversions"}, {"DiagnosticId": "AZ_PY_8", "Text": "list API list_rendering_sessions should return ItemPaged or AsyncItemPaged instead of ItemPaged[RenderingSession]", "TargetId": "azure.mixedreality.remoterendering.RemoteRenderingClient.list_rendering_sessions"}, {"DiagnosticId": "AZ_PY_9", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.RenderingSession.__init__"}, {"DiagnosticId": "AZ_PY_10", "Text": "Typehint is missing for method __init__", "TargetId": "azure.mixedreality.remoterendering.aio.RemoteRenderingClient.__init__"}], "PackageName": "azure-mixedreality-remoterendering"}
```

Comment:
This looks like the APIView token file. It should not be committed to this or any repo.